packages/create-cloudflare/src/common.ts

```diff
@@ -417,8 +440,8 @@ const createCommitMessage = async (ctx: PagesGeneratorContext) => {
 	const details = [
 		{ key: "C3", value: `create-cloudflare@${version}` },
 		{ key: "project name", value: ctx.project.name },
-		{ key: "framework", value: ctx.framework.name },
-		{ key: "framework cli", value: getFrameworkCli(ctx) },
+		...(isPages ? [{ key: "framework", value: ctx.template.id }] : []),
+		...(isPages ? [{ key: "framework cli", value: getFrameworkCli(ctx) }] : []),
```

Comment:
Is it not the case that we can have a framework that is not "pages"?
For example Hono?
In which case we might want to log that information here?